dynamiqs/utils/operators.py

```diff
@@ -131,6 +188,53 @@ def zero(*dims: int, layout: Layout | None = None) -> QArray:
         return SparseDIAQArray(dims, False, (), diags)
 
 
+def zero_like(
```

Comment:
Why not... I like the singular `zero` which means the "zero" or "null" operator of the Hilbert space (because we pass Hilbert space `dims`, not a shape to be filled with zeros). But if you much prefer the plural I'm fine. In another PR to avoid mixing unrelated changes.